### DISCUSSION TITLE
Accesso Spid

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#702] [Richieste di aiuto]
Salve, come si può configurare l'accesso spid?

**Comment by trinko** [comment]
L'accesso SPID è configurabile normalmente in fase di installazione iniziale. 
Si può anche effettuare successivamente, con qualche piccola modifica, perché attualmente non è presente nel menu di configurazione.

Comunque la cosa essenziale è prima seguire le indicazioni dell'AgId, come indicato nella guida, altrimenti non sarà possibile effettuare i test. 
Vedi: [Configurazione SPID](https://iisgiua.github.io/giuaschool-docs/configure.html#configurazione-dello-spid)
